lib/wallet/bitcoin_tx_detail.dart

```diff
@@ -15,6 +17,23 @@ class BitcoinTxDetail extends StatelessWidget {
     final formatter = NumberFormat();
     formatter.minimumFractionDigits = 0;
     formatter.maximumFractionDigits = 0;
+
+    String explorerUrl = "";
+    switch (api.network()) {
```

Comment:
🙃 Might be easier and more maintainable to let the backend give us the URL directly.